core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -191,9 +191,15 @@ object ConsoleConsumer extends Logging {
       .withRequiredArg
       .describedAs("topic")
       .ofType(classOf[String])
-    val whitelistOpt = parser.accepts("whitelist", "Regular expression specifying whitelist of topics to include for consumption.")
+    val whitelistOpt = parser.accepts("whitelist",
+      "DEPRECATED, use --include instead; ignored if --include specified. Regular expression specifying list of topics to include for consumption.")
```

Comment:
We could remove `ignored if --include specified` if we don't accept both together.

Reply:
I think with some other deprecated arguments we simply ignore the deprecated if the new one specified. for example 
- `--broker-list` and `--bootstrap-server` in ConsoleProducer, VerifiableConsumer, VerifiableProducer and GetOffsetShell
And the scripts doesn't fail if both deprecated and new one existed.

Reply:
we can probably resolve this, since we decided to ignore deprecated args